nautobot/core/ui/breadcrumbs.py

```diff
@@ -27,6 +28,15 @@ def __str__(self) -> str: ...
 ReverseParams = Union[dict[str, Any], Callable[[Context], dict[str, Any]], None]
 
 
+def context_object_attr(attr_path: str, context_key: str = "object"):
+    """
+    Helper function that creates callable to easily fetch the object from context and then some nested object.
+
+    Useful for composing breadcrumbs, when we need to add `location` in breadcrumbs path.
+    """
+    return lambda context: attrgetter(attr_path)(context[context_key]) if context.get(context_key) else None
```

Comment:
No idea what this does from the code + current docstring :-) Can you add some examples to the docstring?